packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -72,67 +75,71 @@ export default function TranslationView() {
   const referenceGlosses = referenceGlossesQuery.data.data;
   const targetGlosses = targetGlossesQuery.data.data;
 
-  const bookId = parseInt(verse.id.slice(0, 2));
-  const isHebrew = bookId < 40;
+  const { bookId } = parseVerseId(verse.id);
 
+  const isHebrew = bookId < 40;
   return (
     <div className="px-4">
-      <ol className={`flex flex-wrap ${isHebrew ? 'flex-row-reverse' : ''}`}>
-        {verse.words.map((word, i) => {
-          const targetGloss = targetGlosses[i]?.gloss;
-          const isSaving = glossRequests.some(
-            ({ wordId }) => wordId === word.id
-          );
+      <div className='flex flex-col gap-2'>
```

Comment:
How did you decide to add this extra div rather than putting these classes on the parent div?

Reply:
Hmmm, not sure now. Maybe I did it after merge conflicts.